src/main/resources/canvasCourseDelete.properties.template

```diff
@@ -2,4 +2,10 @@ canvas.token=
 canvas.url=
 esb.url=
 api.call.type=
-sis.term.id=
\ No newline at end of file
+# you can define as many terms as possible with sis_term_id and end date of the term.
+term1=2050;06/24/2015
+term2=
+term3=
+termN=
+#The property tells the count of termN. As the termN property can vary.
```

Comment:
Maybe "This property sets the number of term property lines that will be read from the file."